Bindings/Java/Matlab/examples/c3dExport.m

```diff
@@ -0,0 +1,46 @@
+%% Example of using the Matlab-OpenSim class to read in data and do 
+% operations
+
+%% Choose a c3d file
+[filename, path] = uigetfile('*.c3d');
+c3dpath = fullfile(path,filename);
+
+%% Import Java Lib's
+import org.opensim.modeling.*
+
+%% Construct an opensimC3D object with input c3d path
+c3d = opensimC3D(c3dpath);
+
+%% Get some stats...
+% Get the number of marker trajectories
+nTrajectories = c3d.getNumTrajectories();
+% Get the marker data rate
+rMakers = c3d.getRate_marker();
+% Get the number of forces 
+nForces = c3d.getNumForces();
+% Get the force data rate
+rForces = c3d.getRate_force();
+
+% Get Start and end time
+t0 = c3d.getStartTime();
+tn = c3d.getEndTime();
+
+%% Rotate the data 
+c3d.rotateData('x',-90)
```

Comment:
Just wondering, what is a common use case for rotating the data? I presume it usually involves 2 or 3 rotations of +/-90 degrees. If so, should users call this one method multiple times?

It would be helpful to add more documentation about `rotateData` so users know whether they want +90 or -90, for example. Perhaps spelling out a specific example with a lab coordinate system, etc. would help. I apologize if this is already in a different PR.

Reply:
I have never seen a c3d file where I haven't had to transform about X, it is just the mocap standard. I do agree that we could improve the inline help for this, but additional documentation may be hard since it is a Matlab script — you can't put in Doxygen so I guess the only other space would be on Confluence. 

I still feel strongly that we should ultimately move this feature (rotating Vec3 tables) to the OpenSim side. You should be able to rotate a table or send a (Vec3) table to a Rotation object to perform the operation.

Reply:
I think it's fine to add the documentation inside opensimC3D.m for now, next to the signature for this function. We could move the documentation around later, if necessary.

> I still feel strongly that we should ultimately move this feature (rotating Vec3 tables) to the OpenSim side. You should be able to rotate a table or send a (Vec3) table to a Rotation object to perform the operation.

:)

Reply:
> I still feel strongly that we should ultimately move this feature (rotating Vec3 tables) to the OpenSim side. You should be able to rotate a table or send a (Vec3) table to a Rotation object to perform the operation.

Do you mean add a method like: `DataTable<Vec3>::transformColumns(const SimTK::Transform& X)` ?  Utilities could then form Transforms (e.g. Rotation about X-axis) to rotate or translate `Vec3` data uniformly. That should be straightforward to add. Please add a related issue for the feature.

Reply:
>Do you mean add a method like: DataTable<Vec3>::transformColumns(const SimTK::Transform& X)
That sounds great, Ajay! It would offer a significant speed up and also make rotations a lot more simple for users and the GUI! I will start an issue

Reply:
:+1: